internal/provider/provider.go

```diff
@@ -340,6 +340,7 @@ func createGrafanaClient(d *schema.ResourceData) (string, *gapi.Config, *gapi.Cl
 	auth := strings.SplitN(d.Get("auth").(string), ":", 2)
 	cli := cleanhttp.DefaultClient()
 	transport := cleanhttp.DefaultTransport()
+	transport.MaxConnsPerHost = 2
```

Comment:
limiting the amount of concurrent HTTP connections from the provider makes it not overload the API and DB

Reply:
Can you make that a comment in the code?